README.md

```diff
@@ -57,3 +61,47 @@ AUTH0_CLIENT_SECRET='{CLIENT_SECRET}'
 AUTH0_AUDIENCE=
 AUTH0_SCOPE='openid profile'
 ```
+
+### Database Configuration
+
+1.  Create any [prisma-supported database](https://www.prisma.io/docs/concepts/overview) any localhost or online (tested with [Heroku/PostgreSQL](https://www.heroku.com/postgres)).
+
+1.  Update file **.env** with database url information:
+
+```ini
+DATABASE_URL="postgres://user:password@host:5432/db"
+...
+```
+
+## Database Schema Update
+
+Initial apply of the db schema:
+
+```sh
+yarn prisma generate
+yarn prisma db push
+```
+
+On every change of the file **schema.prisma** please run:
+
+```sh
+yarn prisma db push
+```
+
+On every change of the raw database schema please run:
+
+```sh
+yarn prisma db pull
+```
+
+## Contributing
+
+Please refer to style and contribution guidelines for submitting patches and additions.
+
+1.  **Fork** the repo on GitHub
+1.  **Clone** the project to your own machine
+1.  **Commit** changes to your own branch
+1.  **Push** your work back up to your fork
+1.  Submit a **Pull request** so that we can review your changes
+
+NOTE: Be sure to merge the latest from "upstream" before making a pull request!
```

Comment:
I wonder if we can just refer to something like https://github.com/github/docs/blob/main/CONTRIBUTING.md instead? This will help us keep our README.md small. I’m not suggesting a specific link – any will do, as long as it replaces avoidable text.

Reply:
Sure, sounds good.